opl/cluster_read.py

```diff
@@ -247,7 +250,7 @@ def measure(self, requested_info, name, identifier, metric_query, metric_step):
         assert len(response['MetricList'][0]['DataPoints']) > 0, \
             "'DataPoints' needs to be in response"
 
-        points = [ data_point['Value'] for data_point in response['MetricList'][0]['DataPoints'] ]
+        points = [ data_point.get('Value', 0) for data_point in response['MetricList'][0]['DataPoints'] ]
```

Comment:
Hmm, this basically adds a default. Why is the value missing anyway? Because of how we compute statistical measures out of that data, would it be better to just skip that value? Something like this?
```suggestion
        points = [ data_point["Value"] for data_point in response['MetricList'][0]['DataPoints'] if "Value" in data_point ]
```

Reply:
Not sure but I happened to see this problem other day while i was working on `swatch-tally` performance testing. Sure Jan i am okay with the suggestion :)

Reply:
As per the conversion, included log a message if the total available values less than the total availble dataPoints